spec/services/item_change_set_persister_spec.rb

```diff
@@ -221,6 +221,28 @@
     context 'when change set has changed catkey' do
       let(:new_catkeys) { ['367269'] }
 
+      before do
+        change_set.validate(catkeys: [catkey_before], refresh: false)
+        instance.update
+      end
+
+      it 'invokes object client with item/DRO that has new catkey' do
+        expect(Repository).to have_received(:store).with(
+          cocina_object_with(
+            identification: {
+              barcode: barcode_before,
+              catalogLinks: [
+                { catalog: 'symphony', catalogRecordId: catkey_before, refresh: false }
+              ]
+            }
+          )
+        )
+      end
+    end
+
+    context 'when change set has changed refresh' do
```

Comment:
did you perhaps change the wrong description?

Reply:
Indeed I did. Fixed.